scripts/puppeteer.server.js

```diff
@@ -40,6 +40,15 @@ function handleRoster() {
   });
 }
 
+function handleCameraSelect() {
+  webexSDKAdapter.meetingsAdapter.meetingControls['switch-camera'].display(MEETING_ID).subscribe((display) => {
+    const switchCameraSelect = document.getElementById('switch-camera');
+    if(switchCameraSelect.options.length <= 0 && (display && display.options != null)) {
+      display.options.forEach((camera, key)  => {switchCameraSelect[key] = new Option(camera.label, camera.value);})
```

Comment:
let's have a function `populateSelect` or `setSelectOptions`, and use [first(condition)](https://www.learnrxjs.io/learn-rxjs/operators/filtering/first) to populate it only once:

```js
.pipe(first(display => Array.isArray(display.options))).subscribe(display => setSelectOptions(switchCameraSelect, display.options));
```